kuma/wiki/tests/test_templates.py

```diff
@@ -741,6 +741,28 @@ def test_bad_parameters(self):
         response = self.client.get(url)
         eq_(404, response.status_code)
 
+    def test_no_tidied_content(self):
+        """
+        Verify revisions without tidied content show appropriate message.
+        """
+        Revision.objects.filter(
```

Comment:
maybe a comment that you're intentionally bypassing the save signal handler here? In case someone tries to update it later.